test/integration/cloudscale_test.go

```diff
@@ -111,6 +112,28 @@ func DeleteRemainingVolumes() bool {
 	return foundResource
 }
 
+func DeleteRemainingSubnets() bool {
+	foundResource := false
+
+	subnets, err := client.Subnets.List(context.Background())
+	if err != nil {
+		log.Fatalf("Subnets.List returned error %s\n", err)
+	}
+
+	for _, subnet := range subnets {
+		if strings.HasPrefix(subnet.Network.Name, "go-sdk-integration-test") {
+			foundResource = true
+			log.Printf("Found not deleted subnet: %s (%s)\n", subnet.CIDR, subnet.UUID)
+			err = client.Subnets.Delete(context.Background(), subnet.UUID)
+			if err != nil {
+				log.Fatalf("Subnets.Delete returned error %s\n", err)
+			}
+		}
+	}
+
+	return foundResource
+}
+
```

Comment:
I'm not sure if this is a good idea, because it might point to a network that has not been deleted.

But it's pretty much unclear what is better, because both network/subnet might be responsible for dangling resources. So not saying you need to change something here, but we might discuss this.

Reply:
done